boxtree/tree_build.py

```diff
@@ -1096,7 +1100,7 @@ def my_realloc_zeros_nocopy(ary, shape=nboxes_guess):
                         # At this point, the last entry in level_start_box_nrs
                         # already refers to (level + 1).
                         level_start_box_nrs[-4::-1],
-                        level_used_box_counts[-3::-1]):
+                        level_used_box_counts[-3::-1], strict=False):
```

Comment:
This is set to false because of the fancy indexing: the counts end up to be `len + 1`.